doc/api/util.md

```diff
@@ -666,15 +666,26 @@ Deprecated predecessor of `console.log`.
 
 Deprecated predecessor of `console.log`.
 
-### util._extend(obj)
+### util._extend(target, source)
 
 > Stability: 0 - Deprecated: Use [`Object.assign()`] instead.
 
-The `util._extend()` method was never intended to be used outside of internal
-Node.js modules. The community found and used it anyway.
+*Note: The `util._extend()` method was never intended to be used outside of
+internal Node.js modules. The community found and used it anyway. It is
+deprecated and should not be used in new code. JavaScript comes with very
+similar built-in functionality through [`Object.assign()`].*
 
-It is deprecated and should not be used in new code. JavaScript comes with very
-similar built-in functionality through [`Object.assign()`].
+All the keys and their corresponding values from `source` will be added to
+`target`.
+
+```js
+const util = require('util');
+
+console.log(util._extend({}, {'name': 'Node.js'}));
+  // { name: 'Node.js' }
+console.log(util._extend({'name': 'Node', 'lang': 'js'}, {'name': 'Node.js'}));
+  // { name: 'Node.js', lang: 'js' }
+```
```

Comment:
@thefourtheye Do we need an example for deprecated function? its like warn users and teach them how to use it 😄